components/buttons/buttons.css

```diff
@@ -1,5 +1,5 @@
 .btn{
-    padding: 8px 16px;
+    padding: var(--xs) var(--sm);
```

Comment:
you can have more descriptive names for variable, like in this case you can name them like `--space-xs or --space-sm`